src/common/SurgePatch.cpp

```diff
@@ -3571,7 +3597,7 @@ unsigned int SurgePatch::save_xml(void **data) // allocates mem, must be freed b
     }
 
     {
-        TiXmlElement compat("compatability");
+        TiXmlElement compat("compatability"); // TODO: Fix typo for XT2, LOL!
```

Comment:
What is this?

Reply:
It's a typo. The token should have been compat**i**bility, not compat**a**bility :)